imap_processing/hit/l1a/hit_cdf_attrs.py

```diff
@@ -0,0 +1,762 @@
+        units="V",
+        format="I4",
+    ),
+    "preamp_l234b": replace(
+        hit_hk_base_attrs,
+        # validmax=4095,           # Need this info from instrument team
+        catdesc="Preamp L234B Voltage",
+        fieldname="Preamp L234B Voltage",
+        label_axis="Preamp L234B Voltage",
+        units="V",
+        format="I2",
+    ),
+    "temp0": replace(
+        hit_hk_base_attrs,
+        validmin=-25,
+        validmax=57,
+        catdesc="FEE LDO Regulator Mounted on the Board Next to the Low-dropout "
+        "Regulator",
+        fieldname="FEE LDO Regulator",
+        label_axis="Temp0 - FEE LDO regulator",
+        units="C",
+        format="I2",
+    ),
+    "temp1": replace(
+        hit_hk_base_attrs,
+        validmin=-25,
+        validmax=50,
+        catdesc="Primary Heater Mounted on the Board Next to the Primary Heater "
+        "Circuit",
+        fieldname="Primary Heater",
+        label_axis="Temp1 - primary heater",
+        units="C",
+        format="I2",
+    ),
+    "temp2": replace(
+        hit_hk_base_attrs,
+        validmin=-25,
+        validmax=50,
+        catdesc="FEE FPGA Mounted on the Board Next to the FPGA",
+        fieldname="FEE FPGA",
+        label_axis="Temp2 - FEE FPGA",
+        units="C",
+        format="I2",
+    ),
+    "temp3": replace(
+        hit_hk_base_attrs,
+        validmin=-25,
+        validmax=50,
+        catdesc="Secondary Heater",
+        fieldname="Secondary Heater",
+        label_axis="Temp3 - secondary heater",
+        units="C",
+        format="I2",
+    ),
+    "analog_temp": replace(
+        hit_hk_base_attrs,
+        validmin=-25,
+        validmax=50,
+        catdesc="Chassis Temp Mounted on the Analog Board Close to Thermostats, "
+        "Heaters, and Chassis",
+        fieldname="Analog Temp",
+        label_axis="Analog temp",
+        units="C",
+        format="I2",
+    ),
+    "hvps_temp": replace(
+        hit_hk_base_attrs,
+        validmin=-25,
+        validmax=50,
+        catdesc="Board Temp Mounted Inside Faraday Cage in Middle of Board Near "
+        "Connector Side",
+        fieldname="Board Temp",
+        label_axis="HVPS temp",
+        units="C",
+        format="I2",
+    ),
+    "idpu_temp": replace(
+        hit_hk_base_attrs,
+        validmin=-25,
+        validmax=65,
+        catdesc="LDO Temp Mounted on Top of the Low-dropout Regulator",
+        fieldname="LDO Temp",
+        label_axis="IDPU temp",
+        units="C",
+        format="I2",
+    ),
+    "lvps_temp": replace(
+        hit_hk_base_attrs,
+        validmin=-25,
+        validmax=80,
+        catdesc="Board Temp Mounted in the Middle of Board on Opposite Side of "
+        "Hottest Component",
+        fieldname="Board Temp",
+        label_axis="LVPS temp",
+        units="C",
+        format="I2",
+    ),
+    "ebox_3d4vd": replace(
+        hit_hk_base_attrs,
+        validmin=3,  # doc says min 3.1
```

Comment:
I'm curious about what documentation these limits are coming from?
If it is the limits tab in the TLM spreadsheet, we should verify the meaning. Typically, there are yellow and red limits designated in those spreadsheets that the OPS folks use to trigger alarms which send out notifications that there is something on the s/c or instrument that needs to be investigated. I would interpret the limits in the CDF files differently.